ci/provisioning/post_provision_config_nodes.sh

```diff
@@ -73,6 +78,41 @@ dump_repos() {
         done
 }
 
+send_mail() {
+    local subject="$1"
+    local message="$2"
+    echo $message 2>&1 | mail -s $subject -r "$HOSTNAME"@intel.com "$OPERATIONS_EMAIL"
+}
+
+retry_cmd() {
+    local attempt=0
+    local rc=0
+    while [ $attempt -lt $DAOS_STACK_RETRY_COUNT ]; do
+        if time "$@"; then
+            # Command succeeded, return with success
+            if [ $attempt -gt 0 ]; then
+                send_mail "Command retry successful after $attempt retries" "Command: time $@"
+            return 0
+        fi
+        # Command failed, retry
+        rc=${PIPESTATUS[0]}
+        (( attempt++ ))
+        if [ $attempt -gt 0 ]; then
+          sleep "$DAOS_STACK_RETRY_DELAY_SECONDS"
+        fi
+    done
```

Comment:
(lint) Expected 'fi' matching previously mentioned 'if'. [SC1047]
(lint) Unexpected keyword/token. Fix any mentioned problems and try again. [SC1072]